lib/decorator.js

```diff
@@ -18,8 +18,9 @@ module.exports = function cachedComponent(cacheKeyFn) {
 
     CachedComponent.cacheKeyFn = cacheKeyFn || JSON.stringify;
     CachedComponent.canCache = true;
+    CachedComponent.cacheTtlInSeconds = opts.cacheTtlInSeconds;
     CachedComponent.wrappedComponentName = wrappedComponentName;
-    CachedComponent.displayName = 'CachedComponent(' + wrappedComponentName + ')';
+    CachedComponent.displayName = `CachedComponent(${wrappedComponentName})`;
```

Comment:
the only problem i had with this is that we're not transpiling this lib at all (most of it runs in node anyway) and if a client then requires this file, they will have to specifically make sure they compile it separately. not sure it's worth it just for this change tbh